### PR TITLE
Feature - object detection

### PR DESCRIPTION
**Pull request by st-tuanmai**

### CODE REVIEW

File: examples/object_detection/ios/RunScripts/download_models.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+set -ex
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+MOBILENETV1_SSD_URL="https://storage.googleapis.com/download.tensorflow.org/models/tflite/task_library/object_detection/ios/lite-model_ssd_mobilenet_v1_1_metadata_2.tflite"
+EFFICIENTDET_LITE0_URL="https://storage.googleapis.com/download.tensorflow.org/models/tflite/task_library/object_detection/ios/lite-model_efficientdet_lite0_detection_metadata_1.tflite"
+EFFICIENTDET_LITE1_URL="https://storage.googleapis.com/download.tensorflow.org/models/tflite/task_library/object_detection/ios/lite-model_efficientdet_lite1_detection_metadata_1.tflite"
+EFFICIENTDET_LITE2_URL="https://storage.googleapis.com/download.tensorflow.org/models/tflite/task_library/object_detection/ios/lite-model_efficientdet_lite2_detection_metadata_1.tflite"
+
+MOBILENETV1_SSD_NAME="ssd_mobilenet_v1.tflite"
```
Comment (PaulTR) [inline examples/object_detection/ios/RunScripts/download_models.sh:24]: Comment probably got lost from when you closed the PR on the other repo - if we aren't using all of these models, please don't download them.

**Comment by PaulTR** [comment]
Actually looks like you may have all 4 models scaling correctly now? That's cool, so then yeah you can leave those. Definitely get rid of the part of the bottom sheet that shows percentages/confidences though, those are visible on the screen around the detected objects.

**Comment by st-tuanmai** [comment]
@PaulTR I removed result on bottom sheet

**Comment by PaulTR** [comment]
I merged this, and then realized the app title is still TFL Detect. Please send up a PR where you rename it to just "Object Detection"

**Comment by st-tuanmai** [comment]
> I merged this, and then realized the app title is still TFL Detect. Please send up a PR where you rename it to just "Object Detection"

Hi @PaulTR 
I checked on my device, and the app name has been changed, it may be cached on your device, please delete and reinstall.

**Comment by PaulTR** [comment]
@st-tuanmai I fixed it here: https://github.com/PaulTR/interpreter-samples/pull/22/files

**Pull request merged**